apis/core/types/hydra-box/Api.d.ts

```diff
@@ -0,0 +1,32 @@
+declare module 'hydra-box/Api' {
```

Comment:
`.d.ts` don't seem to be linted?

Reply:
Probably not. I would also not care too much. Quite soon I would like these types to go to `DefinitelyTyped`

Reply:
Works for me, I was simply curious about it being the only file with 4 spaces indentation 👍

Reply:
Ah, fixed that actually 🎉